packages/app/linux/flutter/generated_plugins.cmake

```diff
@@ -3,6 +3,7 @@
 #
 
 list(APPEND FLUTTER_PLUGIN_LIST
+  file_selector_linux
```

Comment:
This is really confusing for me, because I was these lines getting added when I built the app using the docker container, but when I build it locally without the container the lines are removed again. I must have some weird caching going on or something like that.

Reply:
That's odd. Did you resolve this?